packages/storefront-lwc/src/modules/commerce/storeFront/storeFront.html

```diff
@@ -0,0 +1,20 @@
+<template>
+    <lwce-router>
+        <commerce-layout>
+            <commerce-header query={query} slot="header"></commerce-header>
+
+            <lwce-route path="/" exact>
+                <commerce-home></commerce-home>
+            </lwce-route>
+            <lwce-route path="/search/:query">
+                <commerce-product-search-results onquerychange={updateQuery}></commerce-product-search-results>
+            </lwce-route>
+            <lwce-route path="/basket">
+                <commerce-basket></commerce-basket>
+            </lwce-route>
+            <lwce-route path="/product/:pid">
+                <commerce-product-detail></commerce-product-detail>
+            </lwce-route>
+        </commerce-layout>
+    </lwce-router>
+</template>
```

Comment:
what will happen if a path shopper entered is not in this list?

Reply:
The default route `<lwce-route default>` will get rendered. Right now it is just h1 text